pkg/db/vulnerability_metadata.go

```diff
@@ -1,20 +1,30 @@
 package db
 
+// VendorMetadata captures non-standard CVSS fields that vendors can sometimes
+// include when providing CVSS information.  This vendor-specific metadata type
+// allows to capture that data for persisting into the database
+type VendorMetadata interface{}
+
 // VulnerabilityMetadata represents all vulnerability data that is not necessary to perform package-to-vulnerability matching.
 type VulnerabilityMetadata struct {
 	ID           string   // The identifier of the vulnerability or advisory
 	RecordSource string   // The source of the vulnerability information
 	Severity     string   // How severe the vulnerability is (valid values are defined by upstream sources currently)
 	Links        []string // URLs to get more information about the vulnerability or advisory
 	Description  string   // Description of the vulnerability
-	CvssV2       *Cvss    // Common Vulnerability Scoring System V2 values
-	CvssV3       *Cvss    // Common Vulnerability Scoring System V3 values
+	Cvss         []Cvss   // Common Vulnerability Scoring System values
 }
 
 // Cvss contains select Common Vulnerability Scoring System fields for a vulnerability.
 type Cvss struct {
+	VendorMetadata VendorMetadata
+	Metrics        CvssMetrics
+	Vector         string // A textual representation of the metric values used to determine the score
+	Version        string // The version of the CVSS spec, for example 2.0, 3.0, or 3.1
+}
+
+type CvssMetrics struct {
```

Comment:
temporal score is missing here

Reply:
well, maybe not... https://github.com/anchore/grype-db-builder/pull/97/files#r633665629

Reply:
Not sure if you need me to do anything here

Reply:
From offline: there is nothing else to do here